releasenotes/1.9.0.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: releasenotes
+title: Release Notes for Version 1.9.0
+date: 2015-03-18 12:00:00
+release_notes: true
+permalink: /releasenotes/1.9.0/
+---
+
+1. [Introduction](#intro)
+1. [Highlights](#highlights)
+1. [API Changes](#api)
+1. [Known Issues](#issues)
+1. [Credits](#credits)
+1. [Test Results](#test)
+
+## <a id="intro"></a>Introduction
+
+You can read the official announcement at [Apache jclouds 1.9.0 released](TODO). You can read the details of the specific JIRA issues addressed in this release at the [JIRA Release Notes](https://issues.apache.org/jira/secure/ReleaseNote.jspa?version=12329006&styleName=Html&projectId=12314430).
+
+To get jclouds, please see the [jclouds installation guide](/start/install/).
+
+## <a id="highlights"></a>Highlights
+
+* Configured the Animal Sniffer plugin to make sure the code is runtime compatible with Java 6. This will help us support Android.
+* Added AWS Frankfurt region
+* Added AWS signature v4 (excluding S3)
+* Added C4 instance types to AWS EC2
+* Refactored the OkHttp driver to use the native API
+* Added a clean way to customize SSL/TLS connections using the OkHttp driver to avoid POODLE.
+* Initial support for Azure Compute
+* Added authentication to the Docker provider
+* Added the ProfitBricks API
+* Added the Shipyard API
+* Added San José and Honolulu regions to CloudSigma
+* Improved Google Compute Engine Compute support
+* Chef has been promoted to the main repo and updated to support Chef 12
+* Improved Google Compute Engine to use AutoValue
+* Support portable setting and getting container and object ACLs (public-read and private)
+* Support portable multi-object delete
+* Improved multi-part upload compatibility with InputStream Payload
+* Add support for user metadata with filesystem provider
+* Add multi-part upload support for generic S3
+
+
+## <a id="api"></a>API Changes
```

Comment:
> Shouldn't we lead with the new features and not the plumbing like API changes?

Good question - I think it depends on who we expect to read release notes. If we think existing users are the most likely ones, breaking API changes that they need to be aware of seems like the most important topic? If new users, New and Cool stuff should come first, I agree...